app/services/symphony_client.rb

```diff
@@ -34,10 +34,13 @@ def login_by_sunetid(sunetid)
 
   # get a session token by authenticating to symws
   def session_token
-    @session_token ||= begin
+    @session_token ||=
+      begin
       response = request('/user/staff/login', json: Settings.symws.login_params, method: :post)
 
       JSON.parse(response.body)['sessionToken']
+      rescue StandardError
```

Comment:
Can we rescue something more specific than `StandardError` here?  What is the thing that actually fails in this scenario?

Reply:
Yes, this can be `JSON::JSONError`